proteus/LinearSolvers.py

```diff
@@ -411,7 +417,11 @@ def __init__(self,L,par_L,
         self.petsc_L = par_L
         self.csr_rep_local = self.petsc_L.csr_rep_local
         self.csr_rep = self.petsc_L.csr_rep
-        self.bdyNullSpace = bdyNullSpace
+        # HACK FOR CLOSED BDY PROBLEM - NEED TO CHANGE
```

Comment:
We may need to put some more thought into this in general. The reason I put a link to the pde in the LinearOperator object is to pass information about the pde, specifically information on the operator's null space. This is going to be an issue for all the new work on projection schemes as well because we'll often be solving the pressure increment/correction poisson problem with a null space.